appserver/deployment/dol/src/main/java/com/sun/enterprise/deployment/util/DOLUtils.java

```diff
@@ -873,4 +889,20 @@ public static String getComponentEnvId(JndiNameEnvironment env) {
 
     return id;
   }
+
+/**
+ * Supports extreme classloading isolation
+ *
+ * @param application
+ * @param className
+ * @return true if the class is white-listed
+ */
+    public static boolean isWhiteListed(Application application, String className) {
+        for (String packageName : Iterables.concat(application.getWhitelistPackages(), SYSTEM_PACKAGES)) {
```

Comment:
Should the system packages be automatically white listed? It may be that somebody had an old jersey dependency or something like that.